umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10/spatial.clj

```diff
@@ -57,6 +59,14 @@
      [:Minimum_Value (:MinimumValue coord)]
      [:Maximum_Value (:MaximumValue coord)]]))
 
+(defn- convert-vertical-spatial-domains
+  "Validate and convert vertical spatial domains to UMM-C v1.10.0"
```

Comment:
This docstring is confusing. You're converting the vertical spatial domains to dif10 format with '_'